webhook-service/src/test/scala/io/renku/webhookservice/hookcreation/project/ProjectInfoFinderSpec.scala

```diff
@@ -41,22 +48,22 @@ class ProjectInfoFinderSpec
     extends AnyWordSpec
     with MockFactory
     with ExternalServiceStubbing
+    with GitLabClientTools[IO]
     with should.Matchers
     with IOSpec {
 
   "findProjectInfo" should {
 
     "return fetched project info if service responds with 200 and a valid body - personal access token case" in new TestCase {
```

Comment:
I think we can combine this and the next test as I think we don't have to test separately the Oauth and Personal token case.